### PR TITLE
fix: Fixes  api word capitalization freeCodeCamp

### LINKED ISSUE #28446: Capitalisation of the word api
In the curriculum page, the apis and Microservices Certification link should be APIs and Microservices Certification.

screenshot from 2018-10-25 12-26-12

* Browser Name: chrome
* Browser Version: 70.0.3538.67 (Official Build) (64-bit)
* Operating System: Ubuntu 18.04 LTS

### PR DESCRIPTION
**Pull request by akhileshh**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

screenshot_2018-12-02 welcome to learn freecodecamp

Closes #28446

### CODE REVIEW

**Comment by raisedadead** [comment]
Hi @akhileshh 

Thanks for the PR there is an already open PR for the same issue.

**Comment by akhileshh** [comment]
Can you elaborate on that? That's not very helpful for someone's first PR. Do you mean no one else can work on this until the existing PR is closed?

**Comment by lynxlynxlynx** [comment]
No, but it would be better to work within that PR, on top of the existing work — if there's more to be done at all.

File: client/src/components/Map/components/Block.js [hunk 1]
```diff
@@ -108,8 +109,7 @@ export class Block extends Component {
   }
 
   render() {
-    const { completedChallenges, challenges, isExpanded, intro } = this.props;
-    const { blockName } = challenges[0].fields;
```
Comment (Bouncey) [inline client/src/components/Map/components/Block.js]: I was never happy with this, felt awkward. Thank you for replacing it!

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Thanks @akhileshh!!

Congratulations on your first accepted Pull Request to the freeCodeCamp repository!

**Comment by akhileshh** [comment]
> 
> 
> No, but it would be better to work within that PR, on top of the existing work — if there's more to be done at all.

Alright, thanks.

> 
> 
> Hi @akhileshh
> 
> Sincerest apologies. You are correct. I meant to close the other PR in favor of this one. I must increase my coffee dosage.
> 
> Thanks for the PR.
> 
> Will review and let you know for changes if any.

No issues. I was confused because this was my first, thought maybe I missed something.

> 
> 
> Thanks @akhileshh!!
> 
> Congratulations on your first accepted Pull Request to the freeCodeCamp repository!

Thanks! Feel free to assign similar stuff to me, I enjoyed working on this.
What seemed like a simple bug turned out to be a tricky one and I ended up going through a lot of components to understand the flow.

**Pull request merged**